connectors/uplynk/src/main/java/com/theoplayer/android/connector/uplynk/UplynkConnector.kt

```diff
@@ -21,7 +21,7 @@ internal const val TAG = "UplynkConnector"
  * @see [Uplynk Media Platform](https://docs.edgecast.com/video/index.html)
  */
 class UplynkConnector(
-    private val theoplayerView: THEOplayerView,
+    private val theoplayerView: THEOplayerView, private val uplynkConfiguration: UplynkConfiguration
```

Comment:
Nit-pick: put each property on a separate line.
```suggestion
    private val theoplayerView: THEOplayerView,
    private val uplynkConfiguration: UplynkConfiguration
```